components/ContributeProject.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Project, GitHubColors } from '../util';
+import ELink from './ELink';
+
+interface ContributeProjectProps {
+  project: Project,
+  githubColors: GitHubColors
+}
+
+export default function ContributeProject({
+  project,
+  githubColors,
+}: ContributeProjectProps): JSX.Element {
+  const {
+    name,
+    description,
+    repo,
+    lang,
+    topics,
+  } = project;
+
+  return (
+    <>
+      <div className="spaced-row">
+        <h3><ELink link={repo + '/issues'}>{name}</ELink></h3>
+        <p>
+          <span
+            className="dev-language-badge"
+            style={{
+              backgroundColor: githubColors[lang]
+                ? githubColors[lang].color
+                : 'black',
+            }}
+          ></span>{' '}
+          {lang || 'Markdown'}
+        </p>
+      </div>
+      <div>{topics.length > 0 && <span> • {topics.join(', ')}</span>}</div>
```

Comment:
Is there a reason why this • is here? Also, for a new user, they may not know what these things mean; perhaps add a "Topics" heading somewhere.

This is also an interesting use-case for the [`<dl>` tag](https://benmyers.dev/blog/on-the-dl/)!

Reply:
Ooh, this tag seems really cool!